### DISCUSSION TITLE
Mission Statement

### DISCUSSION THREAD

**Discussion by rmpowell77** [bemanproject/beman#3] [Ideas]
We need a solid and easy to articulate mission statement for Beman Project.

https://github.com/beman-org/beman/wiki/Mission-Statement

The Beman Project is dedicated to provide the Guidance and Facilities to develop high quality C++ Libraries with the purpose of making it into the Standard.

Thoughts?

**Comment by rmpowell77** [comment]
Need to add statements/wording about:

Goal is to get Wide Usage by the community.

**Comment by rmpowell77** [comment]
Check out the Boost's missions statements.

**Comment by dustingooding** [comment]
Re: word smithing, I read "The Advantage" recently and it had a section on writing mission statements.  Might be worth keeping in mind.

> What leaders must do to give employees the clarity they need is agree on the answers to six simple but critical questions and thereby eliminate even small discrepancies in their thinking. None of these questions is novel per se. What is new is the realization that none of them can be addressed in isolation; they must be answered together. Failing to achieve alignment around any one of them can prevent an organization from attaining the level of clarity necessary to become healthy. These are the six questions: 1. Why do we exist? 2. How do we behave? 3. What do we do? 4. How will we succeed? 5. What is most important, right now? 6. Who must do what? If members of a leadership team can rally around clear answers to these fundamental questions—without using jargon and shmarmy language—they will drastically increase the likelihood of creating a healthy organization. This may well be the most important step of all in achieving the advantage of organizational health.

**Comment by rmpowell77** [comment]
Some thoughts: I think that this is going to be answered in both the mission statement and the Governance docs.

1. Why do we exist?
To provide the guidance and facilities to ensure the highest quality libraries in ISO standard C++

2. How do we behave?

In the Governance docs, outlines the way we operate

3. What do we do?

We help library authors with development best practices, peer review, and contributing to the C++ standard. 

4. How will we succeed?

We succeed when more libraries are in the standard.

5. What is most important, right now?

Establishing the process to help library authors.  This should probably be in a Roadmap?

6. Who must do what?

Governance doc may help here?

**Comment by rmpowell77** [comment]
The final proposal:

The Beman Project's mission is to *support the efficient design and adoption of the highest quality C++ Standard libraries* through implementation experience, user feedback, and technical expertise.
